target/scripts/startup/setup-stack.sh

```diff
@@ -398,8 +398,8 @@ function _setup_postgrey
 {
   _log 'debug' 'Configuring Postgrey'
 
-  sed -i -E \
-    's|, reject_rbl_client zen.spamhaus.org$|, reject_rbl_client zen.spamhaus.org, check_policy_service inet:127.0.0.1:10023|' \
+  sedfile -i -E \
+    's|(smtpd_recipient_restrictions.*)|\1, check_policy_service inet:127.0.0.1:10023|' \
```

Comment:
I like this. Even more future safe:

```suggestion
    's|(^smtpd_recipient_restrictions =.*)|\1, check_policy_service inet:127.0.0.1:10023|' \
```